### PR TITLE
Doc 10251-Update-Replication-Page

### PR DESCRIPTION
**Pull request by dinaabennett**
Update Replication page with information for Scopes and Collections

### CODE REVIEW

File: modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc [hunk 1]
```diff
@@ -83,6 +83,44 @@ ifdef::is-android[]
...
+When upgrading Couchbase Lite to 3.1, the existing documents in the database will be automatically migrated to the default collection. 
+
+For backward compatibility with the code prior to 3.1, when you set up the replicator with the database, the default collection will be set up to sync with the default collection on Sync Gateway.
+
+.Sync Couchbase Lite database with the default collection on Sync Gateway
+image:ROOT:cbl-replication-scopes-collections-1.png[Sync Couchbase Lite database with the default collection on Sync Gateway,500,,align="left"]
+ 
+
+.Sync Couchbase Lite default collection with default collection on Sync Gateway
+image:ROOT:cbl-replication-scopes-collections-2.png[Sync Couchbase Lite default collection with default collection on Sync Gateway,500,,align="left"]
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc:111]: I think this section can be either removed or change the diagram by adding `Collections: _default` under the `Target:`

File: modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc [hunk 2]
```diff
@@ -83,6 +83,44 @@ ifdef::is-android[]
...
+
+
+=== User-Defined Collections
+
+The user-defined collections specified in the Couchbase Lite replicator setup must exist (and be identical) on the Sync Gateway side to sync. 
+
+.Syncing scope with user-defined collections. Couchbase Lite has fewer collections than the Sync Gateway configuration (no collection filters)
+image:ROOT:cbl-replication-scopes-collections-3.png["Syncing scope with user-defined collections. Couchbase Lite has fewer collections than the Sync Gateway configuration (no collection filters)",500,,align="left"]
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc]: This diagram is not correct. As I have already mentioned in the Google Doc that CBL 3.1 doesn't allow to specify only scope to have all collection in the scope synced. The diagram need to specify collection A and C. 

- Remove the description "All collections in Sample ...."
- Add "Collections: A, C" under "Target: ..."

File: modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc [hunk 3]
```diff
@@ -83,6 +83,44 @@ ifdef::is-android[]
...
+
+.Syncing scope with user-defined collections. Couchbase Lite has more collections than the Sync Gateway configuration (with collection filters)
+image:ROOT:cbl-replication-scopes-collections-4.png["Syncing scope with user-defined collections. Couchbase Lite has more collections than the Sync Gateway configuration (with collection filters)",500,,align="left"]
+
```
Comment (pasin) [inline modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc:123]: In the diagram, change "Filter: C" to "Collections: A"

**Review by pasin — CHANGES_REQUESTED** [review]

**Review by pasin — APPROVED** [review]

**Review by osfameron — APPROVED** [review]
🥳

**Pull request merged**